net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,197 @@
+    make prefix=/app LDFLAGS=-L/app/lib \
+         fluid
+  - |
+    export IEMGUI_FLATPAK_LINKER_FLAGS="-Wl,--export-dynamic -shared" && \
+         make prefix=/app LDFLAGS=-L/app/lib \
+         iemgui
+  - |
+    export MOOCOW_FLATPAK_CONFIGURE="--host=$(uname -m)-pc-linux-gnu" && \
+         make prefix=/app \
+         moocow
+  - |
+    export OSCX_FLATPAK_CONFIGURE_FLAGS="--host=$(uname -m)-pc-linux-gnu" && \
+         export OSCX_FLATPAK_LIBS="-ltirpc" && \
+         export OSCX_FLATPAK_LDFLAGS="-L/app/lib" && \
+         export OSCX_FLATPAK_INCLUDES="-I/app/include/tirpc" && \
+         make prefix=/app \
+         oscx
+  - |
+    make prefix=/app LUA_LIBS="-L/app/lib -llua" \
+         pdlua
+  # now let's install them
+  - |
+    make DESTDIR=/ prefix=/app \
+         adaptive_install arraysize_install autotune_install bassemu_install \
+         boids_install bsaylor_install \
+         comport_install creb_install cxc_install cyclone_install \
+         disis_install earplug_install ekext_install ext13_install fftease_install \
+         flatgui_install fluid_install freeverb_install gem_install ggee_install \
+         hcs_install iem_ambi_install iem_bin_ambi_install iemlib_install \
+         iemguts_install iem_adaptfilt_install iemmatrix_install iemxmlrpc_install \
+         iem_delay_install iem_roomsim_install iem_spec2_install iem_tab_install \
+         iemgui_install jasch_lib_install loaders-libdir_install lyonpotpourri_install \
+         mapping_install markex_install maxlib_install mjlib_install \
+         moocow_install moonlib_install motex_install mrpeach_install oscx_install \
+         pan_install pdcontainer_install pddp_install pdlua_install pdogg_install \
+         plugin_install pmpd_install rjlib_install sigpack_install smlib_install \
+         tof_install unauthorized_install vbap_install windowing_install zexy_install
+
+  sources:
+  - type: git
+    url: https://github.com/pd-l2ork/pd-l2ork/
+    tag: '20231013'
+    commit: 2c3bc68b28ad55e17c07d23a715ef7ff3128068f
+
+- name: pd-abstractions
+  buildsystem: simple
+  subdir: abstractions
+  build-commands:
+  - make DESTDIR=/ prefix=/app install
+  sources:
+  - type: git
+    url: https://github.com/pd-l2ork/pd-l2ork/
+    tag: '20231013'
+    commit: 2c3bc68b28ad55e17c07d23a715ef7ff3128068f
+
+- name: pd-l2ork-integration
+  buildsystem: simple
+  build-commands:
+  # apps folder
+  - mkdir -p /app/lib/pd-l2ork/apps
+  - cp -rf apps/* /app/lib/pd-l2ork/apps/
+  # K12 externals
+  - mkdir -p /app/lib/pd-l2ork/extra/K12
+  - cp -rf l2ork_addons/K12/* /app/lib/pd-l2ork/extra/K12/
+  # install nw
+  - mkdir -p /app/lib/pd-l2ork/bin/nw
+  - tar -x -f nwjs.tar.gz -C /app/lib/pd-l2ork/bin/nw --strip-components=1
+  # wrapper script
+  - install -Dm755 packages/linux_flatpak/pd-wrapper.sh /app/bin
+  # .desktop file (so we appear in list of applications)
+  - install -Dm644 packages/linux_flatpak/net.pdl2ork.PdL2Ork.desktop -t /app/share/applications/
```

Comment:
You should fix these, it won't cause the build to fail but will generate warnings

```
$ desktop-file-validate net.pdl2ork.PdL2Ork.desktop

net.pdl2ork.PdL2Ork.desktop: hint: value "Audio;AudioVideo;Development;Midi;Music;3D" for key "Categories" in group "Desktop Entry" contains more than one main category; application might appear more than once in the application menu
net.pdl2ork.PdL2Ork.desktop: error: value "Audio;AudioVideo;Development;Midi;Music;3D" for key "Categories" in group "Desktop Entry" contains an unregistered value "3D"; values extending the format should start with "X-"
```